pip_audit/dependency_source/resolvelib/pypi_provider.py

```diff
@@ -111,6 +122,37 @@ def get_metadata_for_wheel(url):
     return EmailMessage()  # pragma: no cover
 
 
+def get_metadata_for_sdist(url):
+    data = requests.get(url).content
```

Comment:
This HTTP request can fail, so let's do `raise_for_status` or something similar to propagate it as an exception that we catch and transform into one of our own exceptions.

Reply:
da69505d7d5c657c6e28ae69f605693e8067f06d